### ISSUE TITLE
Improve https://ocaml.org/meetings/  page

### ISSUE THREAD

**Issue by nibble0101** [ocaml/v2.ocaml.org#1315]
Currently when you navigate to the [meetings page](https://ocaml.org/meetings/), there are strange  characters in the text under the `Ocaml meetings > In Europe` section. Check the screenshot below at the end of the lines.

![image](https://user-images.githubusercontent.com/52580190/112962666-4c17bc80-914f-11eb-81f0-a711e02f0b30.png)

When you switch to the French page and translate it to English, you will see the header `Upcoming Events` having an event which was held last year. Check the screenshot below.

![image](https://user-images.githubusercontent.com/52580190/112963431-0a3b4600-9150-11eb-95db-c60efd94dc3b.png)

Further down still on the French page, you will see`Ocaml meetings and meetings`. It is worth pointing out that this is a translation from French to English.  It could therefore be a **mis-translation**. Check the screenshot below.

![image](https://user-images.githubusercontent.com/52580190/112963844-70c06400-9150-11eb-88cd-46703cc14817.png)

When you compare the English and French page, they look quite different and there seems to be differences in content and styling. In my opinion the French page looks better with the side navigation.

**Comment by nibble0101** [comment]
@patricoferris  and @avsm. Could this be within the scope of #1245 or `good first issue`?

PS. I am not an applicant

**Comment by gurleennsidhuu** [comment]
Hi. Can you please help me understand this a little more? Do we have to add a side navigation to the English page + remove the extra characters at the end?

**Comment by nibble0101** [comment]
@gurleennsidhuu  It is great that you have registered interest in working on the issue. My opinion is yes but it is advisable to wait for what @patricoferris  or @avsm  thinks before you start working on it.  Nevertheless it shouldn't stop you from exploring.

I think the following are worth fixing:
1. Remove the stray characters
2. Transfer the outdated event under upcoming events to past events 
3. Make the English page look like French page if it doesn't become too much for a `good first issue`

**Comment by patricoferris** [comment]
Thanks again @nibble0101 your contributions here have been nothing short of superb! 

@gurleennsidhuu feel free to tackle this issue, to keep it manageable I would focus on (1) and (2) for now unless you feel capable to handle the translations.

**Comment by gurleennsidhuu** [comment]
Hi. Does this look fine?
(1) 
![meet](https://user-images.githubusercontent.com/64313783/113288164-356a9480-930c-11eb-8ddc-1a9add424a92.png)
(2) 
![past](https://user-images.githubusercontent.com/64313783/113288180-3bf90c00-930c-11eb-9ef2-c5640d45fc4a.png)
I left the "Upcoming events" column empty. Should I delete the heading or leave it empty?

**Comment by gurleennsidhuu** [comment]
Can I create a new issue for making the English page look like the French page?

**Comment by gurleennsidhuu** [comment]
Hi @patricoferris and @avsm. Looking forward to your response. :)

**Comment by pitag-ha** [comment]
@gurleennsidhuu , have you already worked on (1) and (2)?

**Comment by gurleennsidhuu** [comment]
> @gurleennsidhuu , have you already worked on (1) and (2)?

Yes.

> Hi. Does this look fine?
> (1)
> ![meet](https://user-images.githubusercontent.com/64313783/113288164-356a9480-930c-11eb-8ddc-1a9add424a92.png)
> (2)
> ![past](https://user-images.githubusercontent.com/64313783/113288180-3bf90c00-930c-11eb-9ef2-c5640d45fc4a.png)
> I left the "Upcoming events" column empty. Should I delete the heading or leave it empty?

Can you please confirm if this is correct?

**Comment by gurleennsidhuu** [comment]
Should I keep the 'Prochains evenements' section empty or should I remove it?

**Comment by pitag-ha** [comment]
Maybe we could do it as on the English page? Do you speak French?

**Comment by gurleennsidhuu** [comment]
> Maybe we could do it as on the English page? Do you speak French?

I think the English page doesn't have past and upcoming events section and no, I don't speak or know French.

**Comment by pitag-ha** [comment]
Yeah, I also think in English that's only one section, so we could do the same in French. But I don't know how to tranlsate that to French

**Comment by gurleennsidhuu** [comment]
> Maybe we could do it as on the English page? Do you speak French?

Can we use google translator?

**Comment by pitag-ha** [comment]
I don't think Google translator would a a good idea: we might end up choosing a wrong translation.

**Comment by gurleennsidhuu** [comment]
> I don't think Google translator would a a good idea: we might end up choosing a wrong translation.

I talked to one of my friends who knows French and according to her the google translation 'Atelier utilisateurs et développeurs OCaml' for OCaml Users and Developers Workshop is correct.
